Gemfile

```
@@ -4,15 +4,18 @@ source 'https://rubygems.org'
 
 gem 'nokogiri', '~> 1'
 gem 'puma'
-gem 'rack', '~> 2'
+gem 'rack', '~> 3'
 gem 'rack-cors', require: 'rack/cors'
-gem 'sinatra'
+gem "rackup", "~> 2.2"
```

```suggestion
gem 'rackup', '~> 2.2'
```
Using a mix of two different quotes is a bit tough on my eyes :upside_down_face:

---

Is this all the objections or still looking?

---

I mean I asked so that I can apply all at once but since you had a code suggestion, I just clicked merge. So take your time. no rush.